plugins/test/main_menu_action.test.jsx

```diff
@@ -42,6 +42,7 @@ describe('plugins/MainMenuActions', () => {
                 closeRhsMenu: jest.fn(),
                 unhideNextSteps: jest.fn(),
             },
+            integrationMenuItems: [],
```

Comment:
Where does this property come from? I don't see it anywhere else in this PR

Reply:
From previous changes to add this also to the main menu. Removed.